src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,38 @@
+package tasks;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    public void execute(){
+
+    };
+
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+    };
+
+    public String mark(){
+        isDone = !isDone;
```

Comment:
Will calling mark() on a task that is already marked cause the task to be unmarked?